app/templates/events/eventList.html

```diff
@@ -50,27 +50,33 @@ <h1 class="text-center">Events List for {{selectedTerm.description}}</h1>
 <div class="row">
   <ul class="nav nav-tabs nav-fill mx-2 mb-2" id="pills-tab" role="tablist">
   <li class="col-md-2 col-12 nav-item" role="presentation">
-    <button class="nav-link {{'active' if activeTab == 'studentLedEvents' else ''}}" id="studentLedEvents"
-    data-bs-toggle="pill" data-bs-target="#pills-student-led" type="button" role="tab" aria-controls="pills-student-led" aria-selected="true">Student Led Service</button>
+    <button class="nav-link {{'active' if activeTab == 'volunteerOpportunities' else ''}}" id="volunteerOpportunities"
+    data-bs-toggle="pill" data-bs-target="#pills-volunteer-opportunities" type="button" role="tab" aria-controls="pills-volunteer-opportunities" aria-selected="true">Volunteer Opportunities</button>
   </li>
+ {% if not user.isLaborOnly %}
```

Comment:
What does `isLaborOnly` mean in the context of a user? There's no `isLaborOnly` property in the User class. Am I missing where this is defined?